package.json

```diff
@@ -46,6 +46,7 @@
     "date-fns": "^2.28.0",
     "filter-invalid-dom-props": "^2.1.0",
     "lodash": "^4.17.21",
+    "moment": "^2.29.4",
```

Comment:
@aojin `moment` is deprecated. You can use `date-fns`, which is already imported in this project.

Reply:
@josiasds Did y'all make this change?

Reply:
I'm going to remove this and use `date-fns`.